Core/Core/Network/DownloadManager.swift

```diff
@@ -470,6 +472,60 @@ public class DownloadManager: DownloadManagerProtocol {
             debugLog("SaveFile Error", error.localizedDescription)
         }
     }
+    
+    public func removeAppSupportDirectoryDeprecatedContent() {
+        deleteMD5HashedFolders()
+    }
+    
+    private func getApplicationSupportDirectory() -> URL? {
+        let fileManager = FileManager.default
+        do {
+            let appSupportDirectory = try fileManager.url(
+                for: .applicationSupportDirectory,
+                in: .userDomainMask,
+                appropriateFor: nil,
+                create: true
+            )
+            return appSupportDirectory
+        } catch {
+            print("Error getting Application Support Directory: \(error)")
```

Comment:
Please use debugPrint here and in other places where applicable in code that you wrote.